Source/Scene/MetadataClassProperty.js

```diff
@@ -377,22 +633,18 @@ MetadataClassProperty.prototype.unpackVectorAndMatrixTypes = function (value) {
  * @private
  */
 MetadataClassProperty.prototype.packVectorAndMatrixTypes = function (value) {
```

Comment:
Add some unit tests for this, I don't think MetadadataClassProperty tests this function (it gets tested indirectly through MetadataTableProperty)

Reply:
correction, there were unit tests, just not for arrays of vectors. I'll add some.